bambi/models.py

```diff
@@ -174,6 +174,13 @@ def build(self):
                 if not isinstance(t.prior, Prior):
                     scaler.scale(t)
 
+        # For binomial models with n_trials = 1 (most common use case),
+        # tell user which event is being modeled
+        if self.family.name=='binomial' and np.max(self.y.data) < 1.01:
+            event = next(i for i,x in enumerate(self.y.data.flatten()) if x==1)
+            print('NOTE: Modeling the probability that {}==\'{}\''.format(
```

Comment:
We should probably use a `warning` for this rather than printing; see example earlier in the code.